metadata_layer.py

```diff
@@ -182,6 +185,54 @@ def from_contribution(cls, contribution):
             roles=[contribution.role]
         )
 
+
+    def apply(self, destination, replace=None):
+        """  Update the passed-in Contributor with this ContributorData's information.
+
+        :param: destination -- the Contributor object to write this ContributorData 
+        object's metadata to.
+        :param: replace -- Replacement policy (not currently used).
+
+        :return: the possibly changed Contributor object and a flag of whether it's been changed.
+        """
+        made_changes = False
+
+        log = logging.getLogger("Abstract metadata layer")
+        log.debug(u"Applying %r (%s) into %r (%s)", self, self.viaf, destination, destination.viaf)
+
+        existing_aliases = set(destination.aliases)
+        new_aliases = list(destination.aliases)
+        for name in [self.name] + self.aliases:
+            if name != destination.name and name not in existing_aliases:
+                new_aliases.append(name)
+                made_changes = True
+        if new_aliases != destination.aliases:
+            destination.aliases = new_aliases
+            made_changes = True
+
+        if not destination.lc:
```

Comment:
I patterned it on the contributor merge method.  I don't think it would be a problem to change the code.